apps/namadillo/src/App/Settings/CurrencySelectorEntry.tsx

```diff
@@ -0,0 +1,35 @@
+import { CurrencyInfoListItem } from "@namada/utils";
+import clsx from "clsx";
+import { twMerge } from "tailwind-merge";
+
+type CurrencySelectorEntryType = {
+  currency: CurrencyInfoListItem;
+  selected: boolean;
+  onClick: () => void;
+};
+
+export const CurrencySelectorEntry = ({
+  currency,
+  selected,
+  onClick,
+}: CurrencySelectorEntryType): JSX.Element => {
+  return (
+    <li>
+      <button
+        onClick={onClick}
+        className={twMerge(
+          clsx(
```

Comment:
Does it make a difference to call both `twMerge` and `clsx` here?

Reply:
The `twMerge` is necessary to override the `border-transparent` with `border-yellow` in case the selected is true :)